volttron/platform/control.py

```diff
@@ -1388,7 +1513,7 @@ def update_auth(opts):
 
 def add_role(opts):
     auth_file = _get_auth_file(opts.volttron_home)
-    roles = auth_file.read()[2]
+    roles = auth_file.read()[3]
```

Comment:
Going back to the discussion of the proper job of Control vs. AuthFile, should putting the job of getting 'roles' be put on AuthFile instead of Control? I ask because if the internals of AuthFile change, then Control might also have to change with respect to its use of AuthFile.

Reply:
I think that it will be good to reorganize control. That might be best to do as a separate task dedicated to that effort.